pkg/pod/pod.go

```diff
@@ -42,6 +42,13 @@ const (
 
 	// TaskRunLabelKey is the name of the label added to the Pod to identify the TaskRun
 	TaskRunLabelKey = pipeline.GroupName + pipeline.TaskRunLabelKey
+
+	// TektonHermeticEnvVar is the env var we set in containers to indicate they should be run hermetically
+	TektonHermeticEnvVar = "HERMETIC"
```

Comment:
I might have missed a separate convo elsewhere; do we still want this to be `"TEKTON_HERMETIC"` to indicate it's a reserved name?